docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -63,6 +63,16 @@ Keep `menu` the same as sub-directory the file is in. This will help to manage a
 
 Show new menu section in main page by changing `website/layouts/_default/baseof.html` file.
 
+## Logos
+
+If your company is using Thanos in production, we'd love to showcase your company's logo on our page. Add yourself in [`website/data/sponsors.yml`](/website/data/sponsors.yml) like so:
+
+```yml
+- name: My Awesome Company
+  url: https://wwww.company.com
+  logo: company.png
```

Comment:
where do I put my file? Can we also ask for a specific resolution / size to ensure we dont have alot of back and forth?

Reply:
Done I think.